dotnet/src/SemanticKernel.Abstractions/KernelConfig.cs

```diff
@@ -13,36 +14,30 @@ public sealed class KernelConfig
     /// <summary>
     /// Factory for creating HTTP handlers.
     /// </summary>
-    public IDelegatingHandlerFactory HttpHandlerFactory { get; private set; } = new DefaultHttpRetryHandlerFactory(new HttpRetryConfig());
+    public IDelegatingHandlerFactory HttpHandlerFactory { get; private set; } = new NullHttpHandlerFactory();
 
     /// <summary>
     /// Default HTTP retry configuration for built-in HTTP handler factory.
     /// </summary>
-    public HttpRetryConfig DefaultHttpRetryConfig { get; private set; } = new();
+    [Obsolete("Usage of Semantic Kernel internal retry abstractions is deprecated")]
+    public HttpRetryConfig DefaultHttpRetryConfig =>
+        throw new NotSupportedException("Usage of Semantic Kernel internal retry abstractions is deprecated");
```

Comment:
What should developers do who were previously tweaking config settings? We should point them in the right direction.

Reply:
Agree, but currently we don't have a readme file with suggested packages supporting our HttpHandler Resiliency abstractions. 

I will keep track and as soon we have a readme file to relate will update the message with the reference.